package.json

```diff
@@ -8,14 +8,15 @@
     "node": ">=16"
   },
   "type": "module",
+  "types": "./dist/src/webtransport.d.ts",
   "scripts": {
     "start": "node test/echoserver.js",
     "test": "node test/test.js",
-    "install": "node build.js install",
-    "build": "node build.js build",
-    "rebuild": "node build.js rebuild",
-    "rebuild-debug": "node build.js rebuild-debug -D",
-    "build-debug": "node build.js build-debug"
+    "install": "node build.js install && tsc",
```

Comment:
Actually, I would prefer if the `tsc` can go into the `build.js`. The build process is quite complex and therefore I need the build.js, furthermore `&&` fails on windows and would not pass the unit tests.

Reply:
I've added it to `build.js` and also added `await`s to the invocations of `execbuild` which seemed to be missing.

I've also added a `"types"` npm script that just invokes `tsc` in case you want to build the types without building the C++ code first.

Reply:
Looks good..